src/controller/org.controller/index.js

```diff
@@ -728,14 +719,13 @@ router.get('/org/:shortname/id_quota',
   /*
```

Comment:
It would be good to move this up in the file so it is grouped with the other org tagged endpoints - so after PUT /org/{shortname}